api/src/test/java/org/openmrs/api/impl/ProgramWorkflowServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.api.impl;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.openmrs.ProgramAttributeType;
+import org.powermock.modules.junit4.PowerMockRunner;
+import static org.mockito.Mockito.verify;
+
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.db.ProgramWorkflowDAO;
+
+
+@RunWith(PowerMockRunner.class)
+public class ProgramWorkflowServiceImplTest {
```

Comment:
Aren't these tests a duplicate of what you have put in ProgramWorkflowServiceTest.java?

Reply:
let me counter check

Reply:
they seem to be the same but writing using different logic, which one should I keep?

Reply:
Which one would you prefer? 😊